[2024-11-20T15:50:56.267+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-11-20T15:50:56.367+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: main_dag.database_tasks.check_integrity manual__2024-11-20T02:00:30.526702+00:00 [queued]>
[2024-11-20T15:50:56.374+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: main_dag.database_tasks.check_integrity manual__2024-11-20T02:00:30.526702+00:00 [queued]>
[2024-11-20T15:50:56.375+0000] {taskinstance.py:2865} INFO - Starting attempt 2 of 3
[2024-11-20T15:50:56.392+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): database_tasks.check_integrity> on 2024-11-20 02:00:30.526702+00:00
[2024-11-20T15:50:56.395+0000] {standard_task_runner.py:72} INFO - Started process 65977 to run task
[2024-11-20T15:50:56.397+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'main_dag', 'database_tasks.check_integrity', 'manual__2024-11-20T02:00:30.526702+00:00', '--job-id', '54', '--raw', '--subdir', 'DAGS_FOLDER/main_dag.py', '--cfg-path', '/tmp/tmp5ymrdb6s']
[2024-11-20T15:50:56.398+0000] {standard_task_runner.py:105} INFO - Job 54: Subtask database_tasks.check_integrity
[2024-11-20T15:50:56.442+0000] {task_command.py:467} INFO - Running <TaskInstance: main_dag.database_tasks.check_integrity manual__2024-11-20T02:00:30.526702+00:00 [running]> on host 7331716c13db
[2024-11-20T15:50:56.510+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='veil' AIRFLOW_CTX_DAG_ID='main_dag' AIRFLOW_CTX_TASK_ID='database_tasks.check_integrity' AIRFLOW_CTX_EXECUTION_DATE='2024-11-20T02:00:30.526702+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-11-20T02:00:30.526702+00:00'
[2024-11-20T15:50:56.511+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-11-20T15:50:56.524+0000] {logging_mixin.py:190} INFO - Connection to PostgreSQL DB successful
[2024-11-20T15:50:56.525+0000] {logging_mixin.py:190} INFO - /app/***/dags/database/integrity_check.py
[2024-11-20T15:50:56.902+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-11-20T15:50:56.913+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-11-20T15:50:56.913+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=main_dag, task_id=database_tasks.check_integrity, run_id=manual__2024-11-20T02:00:30.526702+00:00, execution_date=20241120T020030, start_date=20241120T155056, end_date=20241120T155056
[2024-11-20T15:50:56.949+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-11-20T15:50:56.963+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-11-20T15:50:56.965+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
